include/mgmt/mcumgr/smp.h

```diff
@@ -85,6 +85,14 @@ struct zephyr_smp_transport {
 	zephyr_smp_transport_get_mtu_fn *zst_get_mtu;
 	zephyr_smp_transport_ud_copy_fn *zst_ud_copy;
 	zephyr_smp_transport_ud_free_fn *zst_ud_free;
+
+#ifdef CONFIG_MCUMGR_SMP_PKASM
```

Comment:
The thing is that, if we want to be consistent with the Bluetooth spec, we should have:

- SMP Segmentatin
- SMP Reassembly

Because the terms "Fragmentation" and  "Recombination" are used in Bluetooth for low-level breaking up in little chunks and recombining them together. In the case of SMP, this happens at a higher layer, so "Segmentation" and "Reassembly" sounds like a better choice.